### ISSUE TITLE
ATM8 1.19.2 RFTools tablet+storage scanner modules: changing slots empties the tablet bug

### ISSUE THREAD

**Issue by smokeypie27** [AllTheMods/ATM-8#1412]
So, I am playing on a 1.19.2 (ATM1.0.16) and started getting into RFTools. Made a tablet, three storage scanners for each of my bases main buildings with storages, which is about 4-5 chunks away from each other. Connected them to the tablet and everything worked just fine for some time. Then, when I tried to change from one slot to another in the tablet (from one module to another), the tablet emptied itself, changed it's name to the name of my shield in my offhand and all three of my modules dissappeared forever. I tried to make a new one and it kept happening with other ones as well. Tried having an empty offhand or empty inventory and it kept happening. Tried quitting to the menu - same, resetting the game - same, even tried resetting the curseforge and computer. The tablet keeps emptying when changing the slots, even when the storage is in the same chunk or with three different tablets for each one, wih chunks force loaded with FTB or not. Very sad as it seems I am going to have to stop using the tablets. This is my first time playing ATM or using RFTools so maybe I didn't understand the mechanics and broke it myself. Hoping to at least understand where the problem lies.

TLDR: RFTools tablets emptying when changing slots with three storage scanner modules inside. Nothing helps.

**Comment by smokeypie27** [comment]
So, after backuping and deleting and installing my profile again, the problem resolved itself. I hope this means that it won't happen again. Also guess that I broke it myself, somehow. Anyway, hope if people get this problem, this may help them, as I googled this problem and didn't find anything.

**Comment by smokeypie27** [comment]
Nope, happened again, updated to 1.0.19b, didn't help. honestly panicking a little
